### PR TITLE
enable ipv6 tests for telemetry

### PR DESCRIPTION
**Pull request by howardjohn**
**Please provide a description of this PR:**

### CODE REVIEW

File: tests/integration/tests.mk [hunk 1]
```diff
@@ -123,8 +123,4 @@ test.integration.kube.presubmit: | $(JUNIT_REPORT) check-go-tag
 # In presubmit, this target runs a minimal set. In postsubmit, all tests are run
 .PHONY: test.integration.kube.environment
 test.integration.kube.environment: | $(JUNIT_REPORT) check-go-tag
-ifeq (${JOB_TYPE},postsubmit)
```
Comment (howardjohn) [inline tests/integration/tests.mk:126]: DO NOT MERGe. for testing

**Comment by howardjohn** [comment]
Broken because we fetch from GCR in the test: 

```
* cannot fetch Wasm module oci://registry-redirector.registry-redirector-3-92187:1338/istio-testing/wasm/header-injector:latest: could not fetch Wasm OCI image: could not fetch manifest: Get "https://gcr.io/v2/istio-testing/wasm/header-injector/manifests/0.0.1": dial tcp: lookup gcr.io on [fd00:10:96::a]:53: server misbehaving
```

A fix would be to try localhost:5000 first maybe... we can preload the image there?

**Comment by zirain** [comment]
> Broken because we fetch from GCR in the test:
> 
> ```
> * cannot fetch Wasm module oci://registry-redirector.registry-redirector-3-92187:1338/istio-testing/wasm/header-injector:latest: could not fetch Wasm OCI image: could not fetch manifest: Get "https://gcr.io/v2/istio-testing/wasm/header-injector/manifests/0.0.1": dial tcp: lookup gcr.io on [fd00:10:96::a]:53: server misbehaving
> ```
> 
> A fix would be to try localhost:5000 first maybe... we can preload the image there?

that sounds a little hack, when do you plan to load image?

**Comment by howardjohn** [comment]
> > Broken because we fetch from GCR in the test:
> > ```
> > * cannot fetch Wasm module oci://registry-redirector.registry-redirector-3-92187:1338/istio-testing/wasm/header-injector:latest: could not fetch Wasm OCI image: could not fetch manifest: Get "https://gcr.io/v2/istio-testing/wasm/header-injector/manifests/0.0.1": dial tcp: lookup gcr.io on [fd00:10:96::a]:53: server misbehaving
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > A fix would be to try localhost:5000 first maybe... we can preload the image there?
> 
> that sounds a little hack, when do you plan to load image?

idk, was thinking in the script before we run `go test`. But I don't like it either. Probably need a better solution

**Pull request closed without merging**